R/utils-fs.R

```diff
@@ -82,19 +82,30 @@ path_first_existing <- function(...) {
   NULL
 }
 
-path_package_pkgdown <- function(package, ...) {
+path_package_pkgdown <- function(package, bs_version = NULL, ...) {
   if (!requireNamespace(package, quietly = TRUE)) {
     stop(package, " is not installed", call. = FALSE)
   }
 
+  path <- ifelse(
```

Comment:
This should be an inline `if` not an `ifelse` because it’s a scalar, not a vector.